crates/amaru-kernel/src/network.rs

```diff
@@ -12,9 +12,194 @@
+/// ```
+///
+const PREPROD_ERAS: [Summary; 7] = [
+    Summary {
+        start: Bound {
+            time_ms: 0,
+            slot: 0,
+            epoch: 0,
+        },
+        end: Bound {
+            time_ms: 1728000000,
+            slot: 86400,
+            epoch: 4,
+        },
+        params: EraParams {
+            epoch_size_slots: 21600,
+            slot_length: 20000,
+        },
+    },
+    Summary {
+        start: Bound {
+            time_ms: 1728000000,
+            slot: 86400,
+            epoch: 4,
+        },
+        end: Bound {
+            time_ms: 2160000000,
+            slot: 518400,
+            epoch: 5,
+        },
+        params: EraParams {
+            epoch_size_slots: 432000,
+            slot_length: 1000,
+        },
+    },
+    Summary {
+        start: Bound {
+            time_ms: 2160000000,
+            slot: 518400,
+            epoch: 5,
+        },
+        end: Bound {
+            time_ms: 2592000000,
+            slot: 950400,
+            epoch: 6,
+        },
+
+        params: EraParams {
+            epoch_size_slots: 432000,
+            slot_length: 1000,
+        },
+    },
+    Summary {
+        start: Bound {
+            time_ms: 2592000000,
+            slot: 950400,
+            epoch: 6,
+        },
+        end: Bound {
+            time_ms: 3024000000,
+            slot: 1382400,
+            epoch: 7,
+        },
+
+        params: EraParams {
+            epoch_size_slots: 432000,
+            slot_length: 1000,
+        },
+    },
+    Summary {
+        start: Bound {
+            time_ms: 3024000000,
+            slot: 1382400,
+            epoch: 7,
+        },
+        end: Bound {
+            time_ms: 5184000000,
+            slot: 3542400,
+            epoch: 12,
+        },
+
+        params: EraParams {
+            epoch_size_slots: 432000,
+            slot_length: 1000,
+        },
+    },
+    Summary {
+        start: Bound {
+            time_ms: 5184000000,
+            slot: 3542400,
+            epoch: 12,
+        },
+        end: Bound {
+            time_ms: 70416000000,
+            slot: 68774400,
+            epoch: 163,
+        },
+
+        params: EraParams {
+            epoch_size_slots: 432000,
+            slot_length: 1000,
+        },
+    },
+    Summary {
+        start: Bound {
+            time_ms: 70416000000,
+            slot: 68774400,
+            epoch: 163,
+        },
+        end: Bound {
+            time_ms: 89424000000,
+            slot: 87782400,
+            epoch: 207,
+        },
+
+        params: EraParams {
+            epoch_size_slots: 432000,
+            slot_length: 1000,
+        },
+    },
+];
+
+pub fn preprod_era_history() -> &'static EraHistory {
+    static PREPROD_ERA_HISTORY: OnceLock<EraHistory> = OnceLock::new();
+    PREPROD_ERA_HISTORY.get_or_init(|| EraHistory {
+        eras: PREPROD_ERAS.to_vec(),
+    })
+}
```

Comment:
Curious 🤔. Intuitively, I'd have just opted for:

```rust
pub static ERA_HISTORY: LazyLock<EraHistory> = LazyLock::new(|| EraHistory { 
  eras: PREPROD_ERAS.to_vec(),
})
```

Doesn't matter much as this code won't last long.